sensitivity_games/tests/test_point_mass.py

```diff
@@ -97,9 +98,20 @@ def test_dm_curvature_should_be_positive(self):
 
     def test_point_mass_should_converge_to_dare_k(self):
         # TODO: push out T until close, find 'infinite' T
-        np.testing.assert_allclose(self.controller.K.detach(),
-                                   self.optimal_controller.K.detach(),
-                                   atol=zero_tolerance)
+
+        x1 = np.where(self.controller.K.detach() > zero_tolerance,
```

Comment:
suggest adding a comment here to clarify

Reply:
yea, its a hack to drop the small zeros, i.e 1e-03, 1e-07, so that `np.testing.assert_allclose()` works properly. The cleaner way is probably somewhere in the torch docs and has to do with understanding torch tensors better, but this works for now, unless you think of something better.

Reply:
makes sense - I guessed that was what was going on, but I still think a comment in the src is warranted